docs/docs/contributor/server/basics/zapier.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: Zapier App
+sidebar_position: 3
+sidebar_custom_props:
+  icon: TbBrandZapier
+---
+
+## Setup
+- Ask an admin to access the **Twenty 1Password** account
```

Comment:
@martmull this should not go into the public documentation. Here we should provide a way for an external contributor to contribute: create a new Zapier integration, deploy it with the CLI using your own creds...